src/main/java/com/easypost/model/Brand.java

```diff
@@ -0,0 +1,177 @@
+package com.easypost.model;
+
+import com.easypost.net.EasyPostResource;
+
+public class Brand extends EasyPostResource {
+    private String backgroundColor;
+    private String color;
+    private String logo;
+    private String logoHref;
+    private String ad;
+    private String adHref;
+    private String name;
+    private User userID;
```

Comment:
This is a field and is a `String`

Reply:
in our API doc, its User object: https://www.easypost.com/docs/api/node#brand

Reply:
Although the user ID is the id of a `User` object, this is a field that expects/returns a string. Please update it. It should be `string` on our docs as well.